### PR TITLE
Remove frontend checkout

### PR DESCRIPTION
**Pull request by mbj**
Depends on:

* [x] https://github.com/MountainRoseHerbs/spree/pull/670

### CODE REVIEW

**Comment by mbj** [comment]
@dkubb This is ready for review, but not yet for merge.

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -228,17 +228,6 @@ steps:
...
-      docker-wrapper --path frontend --database -- bash -c '
-        ../scripts/setup-ci.sh frontend
-        bundle exec mutant --use rspec --require ./spec/dummy/config/environment --since HEAD~1 -- \
-         Spree::CheckoutController*
-      '
-    agents:
-      queue: mrh-build
-    timeout_in_minutes: 10
-
   - name: ':skull: Spree::OrdersController*'
```
Comment (dkubb) [inline .buildkite/pipeline.yml]: No removal of ` Spree::OrdersController` in this iteration?

Comment (mbj) [inline .buildkite/pipeline.yml]: this is an experimental PR to test if the removal of `spree_frontend` is easy, and if we can expand our iteration as `Spree::OrdersController` is heavily overwritten. I cannot answer this definitively yet. The main PR I'm testing the main app side chainges again is #626

Comment (mbj) [inline .buildkite/pipeline.yml]: Sorry I misunderstood your question, removal of `Spree::OrdersController` is in https://github.com/MountainRoseHerbs/spree/pull/630

File: frontend/app/controllers/spree/orders_controller.rb [hunk 2]
```diff
@@ -65,7 +65,7 @@ def empty
   private
 
     def redirect_update_path
```
Comment (dkubb) [inline frontend/app/controllers/spree/orders_controller.rb]: Can you replace usage of `redirect_update_path` with `cart_path` and remove this method altogether?

Comment (dkubb) [inline frontend/app/controllers/spree/orders_controller.rb]: Can you replace usage of redirect_update_path with cart_path and remove this method altogether?

Comment (mbj) [inline frontend/app/controllers/spree/orders_controller.rb]: @dkubb We need to overwrite it from our main app to inject our checkout hence I left it in place. I do want an entirely different mechanism but we can have that wen we move the orders controller over.

**Comment by dkubb** [comment]
@mbj it looks like there's some merge conflicts to resolve in this PR.

**Comment by mbj** [comment]
> @mbj it looks like there's some merge conflicts to resolve in this PR.

fixed.

File: core/app/helpers/spree/base_helper.rb [hunk 3]
```diff
@@ -11,17 +11,6 @@ def body_class
       @body_class
     end
 
-    def flash_messages(opts = {})
```
Comment (dkubb) [inline core/app/helpers/spree/base_helper.rb]: I think we decided to keep `flash_messages` around since it's used in mrh-spree.

Comment (mbj) [inline core/app/helpers/spree/base_helper.rb]: @dkubb The mrh-spree side companion of this PR removes the call side. I'll double check this before putting this one into `merge-ready`.

**Comment by mbj** [comment]
@dkubb This is ready for the next pass, contains 2 new commits that are needed to make the main app one more smooth (and removes functionality we have there in a superior way to reduce inference).

File: core/app/helpers/spree/base_helper.rb (hunk 3 above)
Comment (mbj) [inline core/app/helpers/spree/base_helper.rb]: The main app one has a superior replacement for this.

**Comment by dkubb** [comment]
@mbj code-wise I am fine with this change. Is it ready to merge? I don't see a "merge-ready" label on it.

**Comment by mbj** [comment]
> @mbj code-wise I am fine with this change. Is it ready to merge? I don't see a "merge-ready" label on it.

The main app side change is not yet ready for review either. With merging this one now we'd commit our selves to do no other change to this repo before that main app one can be merged.

**Comment by mbj** [comment]
Closing this as we have a superior solution in the main app.

**Pull request closed without merging**